applications/plugins/SofaMatrix/src/SofaMatrix/FillReducingOrdering.inl

```diff
@@ -176,4 +221,24 @@ void FillReducingOrdering<DataTypes>::updateElements(
     }
 }
 
+template <class DataTypes>
+void FillReducingOrdering<DataTypes>::updateMesh()
+{
+    // Update topology
+    updateElements(l_topology->getHexahedra(), d_hexahedra);
+    updateElements(l_topology->getTetrahedra(), d_tetrahedra);
+
+    // Update mechanical state
+    auto pos = helper::getWriteOnlyAccessor(d_position);
+    pos.clear();
```

Comment:
I wonder if pos.resize(previousPos.size() then pos[0] ?

Reply:
you are right. Usually, I `reserve` before calling the `push_back`'s, which is equivalent to your suggestion.